src/hooks/useInitializeOnboard.ts

```diff
@@ -96,6 +101,7 @@ export const useInitializeOnboard = (
     try {
       onboard.walletReset()
       Cookies.remove(SELECTED_WALLET_COOKIE_KEY, cookieOptions)
+      config.sentryLog('disconnectWallet2')
```

Comment:
We should check if it was initialized here as well - `if (config.sentryLog) { ... }` or just `config?.sentryLog('disconnectWallet2')`